content/html/reference/elements/blockquote/docs.md

```diff
@@ -18,15 +18,15 @@ recipe: html-element
 ## Short description
 
 The **HTML `<blockquote>` Element** (or *HTML Block Quotation Element*)
-indicates that the enclosed text is an extended quotation. Usually, this
-is rendered visually by indentation (see [Usage notes](#Usage_notes) for how
-to change it). A URL for the source of the quotation may be given using
-the **cite** attribute, while a text representation of the source can be
-given using the
-[`<cite>`](/en-US/docs/Web/HTML/Element/cite)
-element.
-
-## Usage notes
+indicates that the enclosed text is an extended quotation.
+
+## Overview
+
+Usually, this is rendered visually by indentation (see
+[Usage notes](#Usage_notes) for how to change it). A URL for the source of the
```

Comment:
The "Usage notes" parenthetical can be dropped now

Reply:
Good catch! Resolved this, let me know if anything else needs changed.